app/js/enhancedDate.js

```diff
@@ -1,5 +1,99 @@
-/* 
+/*
  * add your module definition for enhancedDate here
  * assign the module to a global variable
- * /
+ */
 
+var enhancedDate = (function iffe() {
+  'use strict';
+  function now() {
+    return new Date();
+  }
+
+  var dateObjects = {
+    dateTime: now(),
```

Comment:
By invoking this here the time is set as soon as the module code is loaded. What may be more flexible would be to only set it once one of the methods on the enhancedDate instance is called.